django/contrib/gis/geos/collections.py

```diff
@@ -16,6 +16,7 @@
 
 class GeometryCollection(GEOSGeometry):
     _typeid = 7
+    _json_type = 'GeometryCollection'
```

Comment:
Do you want to alphabetize the order of the attributes? (looks like that's the current ordering)